libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -170,6 +170,17 @@ const AP_Param::GroupInfo AP_MotorsHeli_RSC::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("GOV_RANGE", 17, AP_MotorsHeli_RSC, _governor_range, AP_MOTORS_HELI_RSC_GOVERNOR_RANGE_DEFAULT),
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    // @Param: AROT_SP
+    // @DisplayName: Autorotation Window Setpoint for External Governor
```

Comment:
Not a blocker but normally we use the word "Target" instead of "set point".  Also if we know it's a PWM or whatever we can use that word instead of either.